sandbox/index.html.example

```diff
@@ -4,19 +4,25 @@
   <meta charset="utf-8" />
   <title>Video.js Sandbox</title>
 
-  <link href="../build/files/video-js.css" rel="stylesheet" type="text/css">
+  <link href="../build/temp/video-js.css" rel="stylesheet" type="text/css">
 
   <!-- LOAD VIDEO.JS SOURCE FILES IN ORDER -->
-  <script src="../build/source-loader.js"></script>
+  <script src="../build/temp/video.js"></script>
```

Comment:
We could theoretically use `grunt watch` but we really do want `watchify`. `watchify` caches the build so rebuilds take minimal amounts of time. That way, you can `watchify` the build and then when you make a change and save you don't have to wait that long before you have to refresh the page.